tscert.go

```diff
@@ -35,10 +35,10 @@ var (
 
 	// TailscaledDialer is the DialContext func that connects to the local machine's
 	// tailscaled or equivalent.
-	TailscaledDialer = defaultDialer
+	TailscaledDialer = DefaultDialer
 )
 
-func defaultDialer(ctx context.Context, network, addr string) (net.Conn, error) {
+func DefaultDialer(ctx context.Context, network, addr string) (net.Conn, error) {
```

Comment:
this isn't the best name imo. And would need docs.

Reply:
It sounds like a type or var. A func would sound like a verb.

Reply:
just `Dial`?  `DialLocalAPI`?

Reply:
went with DialLocalAPI.

Reply:
Sgtm